packages/data-point/lib/reducer-types/reducer-helpers/index.js

```diff
@@ -0,0 +1,68 @@
+const createStub = require('./reducer-stub').create
+
+const reducerAssign = require('./reducer-assign')
+const reducerFilter = require('./reducer-filter')
+const reducerFind = require('./reducer-find')
+const reducerMap = require('./reducer-map')
+const reducerOmit = require('./reducer-omit')
+const reducerPick = require('./reducer-pick')
+
+/*
+"Reducer helpers" are just functions that create reducers.
```

Comment:
Can we omit the word "just" please?

Reply:
done, I also changed some of the other wording in this comment block, like getting rid of the word "compiled"